docs/CONTRIBUTING.md

```diff
@@ -22,4 +22,4 @@ In order to contribute to EDS, please follow this workflow:
 2. Do feature work in accordance to the library's [code guidelines](./CODE_GUIDELINES.md). EDS uses [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), so each commit should follow commit guidelines reflecting the nature of the work.
 3. When work is ready for review, issue a pull request into `next` following the pull request guidelines. Tag `chanzuckerberg/edu-design-system` as reviewers for every PR.
 4. The library owners and trusted contributors will review the work and suggest revisions or approve the pull request
-5. Upon approval, feature gets merged into `next` to be slated for the next library release
+5. Upon approval, feature gets **Squash and merged** into `next` to be slated for the next library release
```

Comment:
Should we update the repository settings to not allow merge commits? https://github.com/chanzuckerberg/edu-design-system/settings

![image](https://user-images.githubusercontent.com/94476616/216200760-fd8cb928-924f-4c29-a093-767bb6c63409.png)

Reply:
Merge commits are still used for merging [release branches](https://github.com/chanzuckerberg/edu-design-system/blob/main/docs/PUBLISHING.md?plain=1#L57) into `main`